Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -55,7 +56,8 @@ class RawPCCProducer : public edm::global::EDProducer<edm::EndLuminosityBlockPro
 
 //--------------------------------------------------------------------------------------------------
 RawPCCProducer::RawPCCProducer(const edm::ParameterSet& iConfig)
-    : putToken_{produces<LumiInfo, edm::Transition::EndLuminosityBlock>(
+    : lumiCorrectionsToken_(esConsumes()),
```

Comment:
This needs to state it will be used at the end LuminosityBlock transition.

Reply:
like this @Dr15Jones ?
```suggestion
    : lumiCorrectionsToken_(esConsumes<edm::Transition:: beginLuminonosityBlock >(),
```

Reply:
ok `1020.0` is running like this, so I'll submit a PR in a second. Interesting that this wasnt picked up by the unit test, I thought https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-5ef42c/20475/unitTests/src/Calibration/LumiAlCaRecoProducers/test/TestCalibrationLumiAlCaRecoProducers/ is running this